packages/ui/src/components/pages/BrowseCommunitiesPage.tsx

```diff
@@ -0,0 +1,45 @@
+import type { ServerPublic } from '~api/router/server/types';
+import { AOHead, Footer, Heading, ServerCard, Navbar } from '../primitives';
+
+const MainContent = ({ servers }: { servers: ServerPublic[] }) => {
+	return (
+		<div className="px-4 2xl:px-[6rem]">
+			<Heading.H1 className="mb-16">Browse Communities</Heading.H1>
+			<div className="grid grid-cols-4 gap-x-4 gap-y-8">
+				{servers.map((server) => {
+					return (
+						<ServerCard
+							server={server}
+							key={`server-${server.id}-area`}
+							type="join"
+						/>
+					);
+				})}
+			</div>
+		</div>
+	);
+};
+
+interface BrowseCommunitiesPageProps {
+	servers: ServerPublic[];
+}
+
+export const BrowseCommunitiesPage = ({
+	servers,
+}: BrowseCommunitiesPageProps) => {
+	return (
+		<div className="mx-auto w-full overflow-x-hidden overflow-y-scroll bg-ao-white scrollbar-hide dark:bg-ao-black">
+			<Navbar />
+			<main className="mb-24 mt-8 bg-ao-white dark:bg-ao-black">
```

Comment:
We need to evaluate our use of the <main> tag throughout the codebase and make sure its not being doubled up, do we use it in the page wrapper?

Reply:
Fixed for this, I think we should discuss this tag futher throughout the codebase